packages/angular_devkit/build_webpack/src/app-shell/schema.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+export interface BuildWebpackAppShellSchema {
+  browserTarget: string;
+  serverTarget: string;
+  renderScript?: string;
```

Comment:
`appModuleBundle` is missing, while `renderScript` is not present in the schema.

Reply:
Done. It's the same flag.